qa/acceptance/spec/shared_examples/cli/logstash-plugin/list.rb

```diff
@@ -0,0 +1,51 @@
+# encoding: utf-8
+require_relative "../../../spec_helper"
+require "logstash/version"
+require "fileutils"
+
+shared_examples "logstash list" do |logstash|
+  describe "logstash-plugin list on #{logstash.host}" do
+    before :all do
+      logstash.install(LOGSTASH_VERSION)
+    end
+
+    after :all do
+      logstash.uninstall
+    end
+
+    context "without a specific plugin" do
+      it "display a list of plugins" do
+        result = logstash.run_command_in_path("bin/logstash-plugin list")
+        # expect(result.exit_status).to eq(0)
+        expect(result.stdout.split("\n").size).to be > 1
+      end
+
+      it "display a list of installed plugins" do
+        result = logstash.run_command_in_path("bin/logstash-plugin list --installed")
+        # expect(result.exit_status).to eq(0)
+        expect(result.stdout.split("\n").size).to be > 1
```

Comment:
can we wrap this in a helper/matcher, something like `expect(logstash). to have_installed_at_least_one_plugin` ? easier to maintain and cleaner, what do you think?